charts/armonik/charts/compute-plane/values.yaml

```diff
@@ -0,0 +1,683 @@
+nameOverride: ""
+replicaCount: 1
+namespace: armonik
+shareProcessNamespace: false
+restartPolicy: Always
+repository: dockerhubaneo
+registry: ""
+imagePullSecrets: []
+
+global:
+  imageRegistry: ""
+
+serviceAccount:
+  create: true
+  name: compute-plane
+  annotations: {}
+  secrets: []
+
+service: {}
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+  # Note: PodSecurityPolicy will not be created when Kubernetes version is 1.25 or later.
+  pspEnabled: false
+
+podDisruptionBudget:
+  # https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+  enabled: false
+  minAvailable:
+  maxUnavailable:
+
+computePlane:
+  metadata:
+    name: "compute-plane"
+    labels:
+      app: "armonik"
+      service: "compute-plane"
+  partition:
+    default:
+      replicas: 1
+      pollingAgent:
+        image: "armonik_pollingagent"
+        tag: "0.31.2"
```

Comment:
Maybe we can have a centralized "core tag" that is just reused as the default tag everywhere in the chart.